### DISCUSSION TITLE
Textbox on Maui .net 6

### DISCUSSION THREAD

**Discussion by GabrieleTorricella** [dotnet/maui#3458] [Q&A] [answered]
is it possible to use the textboxes as it was done with Xamarin / WPF? Let me explain ... I need to insert some textboxes for a Login Form using Maui ... Do you have an example of how to do it? Because at the moment it is not possible for me

**Comment by nogginbox** [comment, marked as answer]
I've been using the ```Entry``` control for users to enter data in forms.

```
<Entry Text="{Binding UserName, Mode=TwoWay}" />
```

Would this work for your use case? I don't think there is a ```TextBox``` control. There is also a ```Label``` control for showing text and an ```Editor``` control for multiline text input.

I've found this page on Xamarin Forms mainly still relevant for MAUI:
https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/text/

**Reply by GabrieleTorricella** [reply]
Thank you for your answer! It helps me a lot!. 
I came from WPF and WebForm applications and I used a lot textbox controllers, so thank you again 'cause I didn't think about Entry controller or something else

**Reply by nogginbox** [reply]
I had a similar problem when I started. You can't google/bingle the answer if they're using a word you're not expecting.

**Reply by GabrieleTorricella** [reply]
Excuse me if I disturb you but... Do you know how can I start with a page (for example, a Login form) and then "navigate" to a home page? 'Cause all of the things I found talk about a Navigate method of System but I won't navigate back to the LoginPage (except if user'll click "logout") but the navigate method do that

**Reply by nogginbox** [reply]
I'm not sure of the best way to do this. I've not done it. You could try using Navigation.RemovePage(loginPage) after you've navigated away from the login page. But there might be a better way. I'd probably start a new question for this, as not many people will notice this as a reply to an answered question.

**Reply by nogginbox** [reply]
You may also want to look at Shell.
